ccvm_simulators/solvers/mf_solver.py

```diff
@@ -398,6 +398,65 @@ def _optics_machine_energy_callable(
 
         return _optics_machine_energy_callable
 
+    def _optics_machine_time(self, machine_parameters: dict = None):
+        """The wrapper function of calculating the average time spent by the
+            solver on a single instance, as if the solving process was to be performed on
+            an optical MF-CCVM machine.
+
+        Args:
+            machine_parameters (dict, optional): Parameters of the optical MF-CCVM
+                machine. Defaults to None.
+
+        Raises:
+            ValueError: when the given machine parameters are not valid.
+            ValueError: when the given dataframe does not contain the required columns.
+
+        Returns:
+            Callable: A callable function that takes in a dataframe and problem size and
+                returns the average average time spent by the solver on a single instance.
```

Comment:
Average wasn't average enough 😂  (I made this mistake a few places)
```suggestion
                returns the average time spent by the solver on a single instance.
```